src/test/shared/sam/debugger/awsSamDebugConfigurationValidator.test.ts

```diff
@@ -42,6 +42,23 @@ function createCodeConfig(): AwsSamDebuggerConfiguration {
     }
 }
 
+function createApiConfig(): AwsSamDebuggerConfiguration {
+    return {
+        type: 'aws-sam',
+        name: 'name',
+        request: 'direct-invoke',
+        invokeTarget: {
+            target: 'api',
```

Comment:
Use `API_TARGET_TYPE`

Reply:
None of the test code in here does that so I followed the convention

Reply:
let's skip that for now. It is not really clear why we need the constants, since we already have defined the type.